store-and-retrieve-data/use-storagehub-sdk/create-a-bucket.md

```diff
@@ -1,4 +1,237 @@
 ---
 title: Create a Bucket
-description: Guide to creating a new storage bucket using StorageHub SDK.
+description: Guide on how to create a new storage bucket with the StorageHub SDK.
 ---
+
+# Create a Bucket
+
+## Prerequisites
+- [Node.js](https://nodejs.org/en/download){target=_blank} v18+ installed
+- Project folder created
+- `package.json` file initialized
+- Typescript and Node type definitions added
+- Created a Typescript config
+- `@storagehub-sdk/core` and `@storagehub-sdk/msp-client` installed 
+See the [Get Started guide](/store-and-retrieve-data/use-storagehub-sdk/get-started.md) for detailed setup instructions.
```

Comment:
```suggestion
See the [Get Started guide](/store-and-retrieve-data/use-storagehub-sdk/get-started) for detailed setup instructions.
```

Reply:
You generally don't need to include the .md when you're referencing pages on the site